src/core/MIGRATION.md

```diff
@@ -156,15 +173,15 @@ Core services that expose functionality to plugins always have their `setup` fun
 
 ### Integrating with other plugins
 
-Plugins can expose public interfaces for other plugins to consume. Like `core`, those interfaces are bound to the lifecycle functions `setup` and/or `stop`.
+Plugins can expose public interfaces for other plugins to consume. Like `core`, those interfaces are bound to the lifecycle functions `setup` and/or `start`.
 
-Anything returned from `setup` or `stop` will act as the interface, and while not a technical requirement, all Elastic plugins should expose types for that interface as well.
+Anything returned from `setup` or `start` will act as the interface, and while not a technical requirement, all Elastic plugins should expose types for that interface as well.
```

Comment:
```suggestion
Anything returned from `setup` or `start` will act as the interface, and while not a technical requirement, all Kibana plugins should expose types for that interface as well.
```

Reply:
Actually I think "Elastic" makes more sense here because we're talking about plugins built by people at Elastic, not all Kibana plugins that exist.

Reply:
ok, although for me it sounds like a recommendation relevant for all Kibana plugins

Reply:
Technically it is a recommendation for 3rd parties, but a requirement for Elastic. I'll update it to make this more clear.